input/docs/writing-builds/reproducible-builds/pinning-cake-version.md

```diff
@@ -20,7 +20,7 @@ To have deterministic builds it is important that on every build the same versio
     <div id="frosting1" class="tab-pane fade">
         <p>
             Make sure to use a fixed version number for <code>Cake.Frosting</code> in the <code>*.csproj</code> file:<br/>
-<pre><code class="language-xml hljs">&lt;PackageReference Include="Cake.Frosting" Version="0.38.4" /&gt;</code></pre>
+<pre><code class="language-xml hljs">&lt;PackageReference Include="Cake.Frosting" Version="1.0.0" /&gt;</code></pre>
```

Comment:
In the meantime we know what the latest version is (same functionality which shows it on home page). Instead of using a fixed version we could use the dynamic latest version

Reply:
I'm in favor of this - I was not sure if the fact that Frosting is currently documented as 0.38.4 and Scripting as 0.38.5 was on purpose or not

Reply:
:white_check_mark: Updated to use latest version of Cake dynamically